module/Decision/src/Decision/Controller/MemberController.php

```diff
@@ -19,6 +20,23 @@ public function indexAction()
     }
 
     /**
+     * Search action, allows searching for members.
+     */
+    public function searchAction()
+    {
+        $name = $this->params()->fromRoute('name');
+        $members = array();
+        foreach ($this->getMemberService()->findMemberByName($name) as $member) {
+            //TODO: this returns a lot of data, much more than is needed in most cases.
```

Comment:
It's better to give too much information here. Because then we can use the same call for multiple search types.